### ISSUE TITLE
Change minimum contig output length

### ISSUE THREAD

**Issue by SorenKarst** [fenderglass/ABruijn#71]
Hello,

Thank you for an excellent tool.

I got a feeling there is a hardcoded minimum length for outputting contigs, and guess this is around 5000 bp. Is this true and if so would it be possible to manually change it? I  have an application where I am interested in contigs around 4500 bp in length.

**Comment by mikolmogorov** [comment]
Hi,

The minimum theoretical contig length is 500, although a high value of the minimum overlap parameter (which is selected automatically within 1000-5000 range) might miss short genomic elements (that are shorter than the parameter, plasmids, for example). You might try to manually set minimum overlap to a lower value. Also, in case you are interested in plasmids (or other kinds of short circular sequences), there is a `--plasmids` mode that is designed to recover them.

**Comment by mikolmogorov** [comment]
Closing this for now - feel free to reopen if you have more questions.

**Comment by SorenKarst** [comment]
That is fine. I tried to adjust the suggested settings, but still no assembly. I think my data type is just not compatible with flye. Thanks for taking the time to help me out.

**Comment by jianshu93** [comment]
Hi both,

I want to filter short assemblies, e.g., shorter than 5kb, how should I do that since I did no see a contig length filtering, option, increasing overlap length to 1000, 2000 or something (assume defaults is 500?).
Thanks,
Jianshu

**Comment by mikolmogorov** [comment]
Flye was not designed for very short assemblies, unfortunately.

**Comment by yujitokumoto** [comment]
Hello Mikhail, I have a question,
I am trying to assemble a plant genome using ONT reads (including R9.4.1 and R10.3).
When I set the parameter of minimum overlap read length to default (auto = 3000bp), I was getting reads shorter than 500bp. We changed the parameter for minimum overlap read length to 5000 and tried to reassemble the genome, but still reads shorter than 500 bp were output (table below).
I wonder why these results are being output? 
The genome size is about 800 Mb and the flye version is 2.9.5.
Total bases used for analyses is 20.54 Gb (N50=10,317) with a coverage of x25.30.

![Image](https://github.com/user-attachments/assets/3ac403c6-fe15-485d-9bf1-4cfe497335b5)
